HtmlMesh/src/pointer-events-capture-behavior.ts

```diff
@@ -114,11 +114,16 @@ const onPointerMove = (evt: PointerEvent) => {
         releaseCurrent();
     }
 
-    // If there is a picked mesh and it is not the current capturing mesh, capture 
-    // the pointer events.  Note that the current capturing mesh has already been 
+    // If there is a picked mesh and it is not the current capturing mesh, capture
+    // the pointer events.  Note that the current capturing mesh has already been
     // released above
     if (pickedMesh) {
-        pointerCaptureBehavior!.capturePointerEvents();
+        /*
```

Comment:
I'm okay with this, but I'm just not sure how pointerCaptureBehavior can be undefined here (though others have reported the same).  The mesh won't be picked if the pointerCaptureBehavior is undefined due to the scene predicate.  I'm concerned with masking a bigger issue.  If you know how this was occuring, or have a repro I can check out, I would be most grateful.

Reply:
Okay, I will identify the cause of the problem in my project and report it to you